# reason:
#   upstream believes Ubuntu/Debian are the only ones
#   to report "$(uname -s)-$(uname -m)" as Linux-x86_64 ...

--- a/Makefile
+++ b/Makefile
@@ -138,14 +138,6 @@
 LIBRARY_REL ?= lib
 PKGCONFIG_REL ?= libdata
 
-else ifeq ($(KERNEL_NAME)-$(MACHINE_NAME), Linux-x86_64)
-
-# default for Debian/Ubuntu x86_64
-BINARY_REL ?= bin
-INCLUDE_REL ?= include
-LIBRARY_REL ?= lib/x86_64-linux-gnu
-PKGCONFIG_REL ?= $(LIBRARY_REL)
-
 else
 
 # NetBSD, ... and Linux64/Linux32 variants that use plain lib directory
